pkg/discovery/globalnet/globalnet.go

```diff
@@ -284,38 +284,40 @@ func isCIDRPreConfigured(clusterID string, globalNetworks map[string]*GlobalNetw
 
 func ValidateGlobalnetConfiguration(subctlData *datafile.SubctlData, netconfig Config) (error, string, uint) {
 	status.Start("Validating Globalnet configurations")
+	var err error
+	var clusterSize uint
```

Comment:
```suggestion
```

The variables don’t need to be defined globally.

Reply:
I created those for CheckStatus(err) which is not required now.